ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py

```diff
@@ -468,3 +468,36 @@ def test_torch_as_tensor(dtype_and_x):
         fn_tree="as_tensor",
         input=input[0],
     )
+
+
+# zeros_like
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    dtypes=_dtypes(),
+    requires_grad=_requires_grad(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.zeros_like"
+    ),
+)
+def test_torch_zeros_like(
+    dtype_and_x,
+    dtypes,
+    requires_grad,
+    device,
+    num_positional_args,
+):
+    dtype, input = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=[False],
```

Comment:
please have a look at the deep dive document （https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html）about writing frontend function tests so that the as_variable_flags, native_array_flags can be set not manually.